api/src/main/resources/workbench.yaml

```diff
@@ -20,30 +26,36 @@ info:
     name: "BSD"
 host: "api.pmi-ops.org"
 securityDefinitions:
-  # Establish the fact that *some endpoints* are OAuth protected
-  # by defining an `aou_oauth` security mode, which we'll assing
-  # to any protected endpoints below.
+#  x-aou-note: |
+#    Establish the fact that *some endpoints* are OAuth protected
+#    by defining an `aou_oauth` security mode, which we'll assing
+#    to any protected endpoints below.
   aou_oauth:
-    # TODO: Vet/fix this token and/or authorization URL to work in practice.
-    # These are currently included simply to satisfy the Swagger specification,
-    # as this is not directly used to dictate oauth details (just used to
-    # annotate which methods require oauth).
-    authorizationUrl: ""
-    tokenUrl: ""
+    x-aou-note: |
+      TODO: Vet/fix this token and/or authorization URL to work in practice.
+      These are currently included simply to satisfy the Swagger specification,
+      as this is not directly used to dictate oauth details (just used to
+      annotate which methods require oauth).
+    authorizationUrl: "https://fake-authorization-url.pmi-ops.org"
+    tokenUrl: "https://fake-token-url.pmi-ops.org"
     type: oauth2
     flow: accessCode
 schemes:
   - "https"
 produces:
   - "application/json"
-# Establish the fact that all endpoints are protected: this annotation
-# ensures that client libraries know to send bearer tokens when calling
 security:
   - aou_oauth: []
-
-# Throughout, we use integer/int64 in preference to string/date-time because Swagger's
-# date formatting is inconsistent between server and client. Time values are stored as
-# milliseconds since the UNIX epoch.
+tags:
+  - name: cron
+    description: |
```

Comment:
Arguably this is an internal note and should move to the `x-aou-note` section or be reworded for the benefit of API consumers.